.github/workflows/deploy.yml

```diff
@@ -134,11 +134,3 @@ jobs:
             -H 'Accept: application/vnd.github.everest-preview+json' \
             -H 'Authorization: token ${{ secrets.HOMEY_GITHUB_ACTIONS_BOT_PERSONAL_ACCESS_TOKEN }}' \
             -d '{"event_type": "update-homey-lib"}'
-             
-      - name: Trigger repository dispatch event athombv/athom-cloud-developer
-        run: |
-          curl -X POST \
-            https://api.github.com/repos/athombv/athom-cloud-developer/dispatches \
-            -H 'Accept: application/vnd.github.everest-preview+json' \
-            -H 'Authorization: token ${{ secrets.HOMEY_GITHUB_ACTIONS_BOT_PERSONAL_ACCESS_TOKEN }}' \
-            -d '{"event_type": "update-homey-lib"}'
```

Comment:
We could make this point to https://github.com/athombv/athom-cloud-driver-reference/
I already added the workflow there: https://github.com/athombv/athom-cloud-driver-reference/blob/master/.github/workflows/update-homey-lib.yml (but its still missing any tokens/permissions of course)

Reply:
Although likely in the future the "driver reference" will likely become part of https://github.com/athombv/node-homey-apps-sdk-v3 but I still need to discuss that with Harwin